pytorch_lightning/core/lightning.py

```diff
@@ -1961,12 +1964,14 @@ def get_from_queue(self, queue: torch.multiprocessing.SimpleQueue) -> None:
 
         Args:
             queue: the instance of the queue from where to get the data.
+
+        .. deprecated:: v1.5
+            This method was deprecated in v1.5 in favor of
+            `DDPSpawnPlugin.get_from_queue`
+            and will be removed in v1.7.
         """
-        # NOTE: `add_to_queue` needs to be called before
-        callback_metrics: dict = queue.get()
-        self.trainer.callback_metrics.update(
-            apply_to_collection(callback_metrics, np.ndarray, lambda x: torch.tensor(x))
-        )
+        if self.trainer and isinstance(self.trainer.training_type_plugin, pl.plugins.training_type.DDPSpawnPlugin):
```

Comment:
there shall be also raised warning

Reply:
this is done in configuration_validator to avoid sending multiple warnings